redun/executors/docker.py

```diff
@@ -79,7 +79,12 @@ def run_docker(
 
     # Environment args.
     common_args.extend(
-        ["-e", "AWS_ACCESS_KEY_ID", "-e", "AWS_SECRET_ACCESS_KEY", "-e", "AWS_SESSION_TOKEN"]
+        [
+            "-e", "AWS_ACCESS_KEY_ID",
+            "-e", "AWS_SECRET_ACCESS_KEY",
+            "-e", "AWS_SESSION_TOKEN",
+            "-e", "AWS_DEFAULT_REGION"
+        ]
```

Comment:
Reformat the list for linters.

```suggestion
        [
            "-e",
            "AWS_ACCESS_KEY_ID",
            "-e",
            "AWS_SECRET_ACCESS_KEY",
            "-e",
            "AWS_SESSION_TOKEN",
            "-e",
            "AWS_DEFAULT_REGION"
        ]
     )
```